.circleci/config-2_1.yml

```diff
@@ -202,6 +217,9 @@ j11_with_dtests_jobs: &j11_with_dtests_jobs
     - j11_jvm_dtests:
         requires:
           - j11_build
+    - j11_cqlshlib_tests:
```

Comment:
should we have an approval here?  j11 doesn't really run anything without explicit approval, this would run for all commits.

Reply:
-1. There is already a ticket I can't find now to remove some j11 approvals.